ci4/app/Config/Logger.php

```
@@ -0,0 +1,151 @@
+<?php
+
+namespace Config;
+
+use CodeIgniter\Config\BaseConfig;
+use CodeIgniter\Log\Handlers\FileHandler;
+use CodeIgniter\Log\Handlers\HandlerInterface;
+
+class Logger extends BaseConfig
```

PHPStan (2.2.2)

[error] Class Config\Logger extends unknown class CodeIgniter\Config\BaseConfig.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)